SolastaUnfinishedBusiness/Behaviors/Specific/_GLBM.cs

```diff
@@ -1014,6 +1014,17 @@ when attacker.UsedSpecialFeatures.ContainsKey(featureDefinition.Name):
                             {
                                 break;
                             }
+                            // Check Light Energy Crossbow Bolt condition is active for Demon Hunter
```

Comment:
Same check is present in the `SmiteSpells2024Context.cs` - move it static method in subclass file and call from here.